### ISSUE TITLE
Can't run scene if no main scene assigned

### ISSUE THREAD

**Issue by cbscribe** [godotengine/godot#17028]
**Godot version:**
3.0.1

**OS/device including version:**
Mac OS 10.12.6

**Issue description:**
"Play Scene" won't play a scene.
No output in the editor, but running from the command line gives:

    Error: Can't run project: no main scene defined.

**Steps to reproduce:**
1. Make a new project
2. Add a node and save the scene
3. Press the "Play Scene" button - scene won't open
4. Make a second scene and assign it as the Main scene
5. "Play Scene" on the first scene works

Basically, no individual scenes will run unless a main scene is assigned.

**Comment by akien-mga** [comment]
That one's for me, probably a regression from my recent `main.cpp` or
`ProjectSettings` fixes.

2018-02-26 7:39 GMT+01:00 Chris Bradfield <notifications@github.com>:
